apps/src/StudioApp.js

```diff
@@ -1810,6 +1810,19 @@ StudioApp.prototype.getUnfilledFunctionalBlockError = function (topLevelType) {
   }
 };
 
+/**
+ * @returns {boolean} True if we have a function or variable named "" (empty string)
+ */
+StudioApp.prototype.hasEmptyFunctionOrVariableName = function () {
+  return Blockly.mainBlockSpace.getTopBlocks().some(function (block) {
+    if (block.type !== 'functional_definition') {
+      return false;
+    }
+
+    return block.getProcedureInfo().name === '';
```

Comment:
First, and this might be a dumb question, are 'functional_definition' blocks always top blocks?